generators/client/templates/angular/src/main/webapp/app/_app.module.ts

```diff
@@ -23,6 +23,7 @@ import { BrowserModule } from '@angular/platform-browser';
 import { Ng2Webstorage } from 'ng2-webstorage';
 
 import { <%=angularXAppName%>SharedModule, UserRouteAccessService } from './shared';
+import {  <%=angularXAppName%>AppRoutingModule} from './app-routing.module';
```

Comment:
This might fail tslint due to extra space

Reply:
Since the CI is green I don't think it will fail. But I'll test right now

Reply:
I do confirm no linting issue.

Reply:
but still can you fix the spacing it looks weird. I guess there is an extra space in all places where `<%=angularXAppName%>AppRoutingModule` used probably a find replace error